src/components/App/App.tsx

```diff
@@ -0,0 +1,65 @@
+import React, {useState, useEffect} from "react";
+// @ts-ignore
+import logo from "./phish-logo.png";
+import { v4 as uuidv4 } from 'uuid';
+import "./App.css";
+
+// @ts-ignore
+import Search from "./Search/Search.jsx";
+import Song from "../../interfaces/Song";
+import GoogliResponse from "../../interfaces/GoogliResponse";
+
+const App = () => {
+  const [query, setQuery] = useState<string>('');
+  const [songs, setSongs] = useState<Song[]>([]);
+
+  const updateSearchQuery = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(e.target.value);
+    search();
+  };
+
+  const search = () => {
+    const filter = query;
+    const uuid = uuidv4();
```

Comment:
Previousl, `uuid` was being tracked in state, but since it is never reset once instantiated, we can set it directly in `const search`.